### ISSUE TITLE
Agenesis of putamen

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5827]
**Preferred term label:**
see various MRI terms here https://pubmed.ncbi.nlm.nih.gov/31412107/

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://pubmed.ncbi.nlm.nih.gov/31412107/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Definition: A developmental defect characterized by the absence of the putamen owing to its failure to develop. [PMID:31412107]

Comment: The basal ganglia are symmetrical subcortical grey nuclei at the core of the extrapyramidal system, composed by the striatum (which comprises the putamen, caudate nucleus, olfactory tubercle, and nucleus accumbens), the pallidum, the substantia nigra, and the subthalamic nucleus. Congenital malformations of the basal ganglia deriving from defects of embryogenesis are very rare. The basal ganglia embryologically derive from the prosencephalon (or forebrain), the most rostral of the three vesicles originating from early patterning of the neural tube along the anterior-posterior axis.

**Comment by pnrobinson** [comment]
Also
Hot cross bun sign
A cruciform-shaped hyperintensity within the pons found on T2-weighted magnetic resonance imaging (MRI).[PMID: 25886304]
Comment: It is commonly associated with atrophy of the pons, cerebellum, and putamen in multiple system atrophy (MSA). In
